entity-server-api/src/main/java/org/terracotta/entity/CommonServerEntity.java

```diff
@@ -46,4 +46,12 @@
    *  respect to all other messages enqueued for the entity.</p>
    */
   void destroy();
+
+  /**
+   * Notify an entity that a particular client has a shared understanding of the state
+   * of the system, such as after the client disappears, or after reconnect has finished.
+   *
+   * @param client client
+   */
+  void notifyClientIsSynched(ClientDescriptor client);
```

Comment:
Okay now I get it. In the reconnect case this method is NOT invoked immediately after the handleReconnect call on the active BUT only after the client is done resending all the messages in its resend queue as well. So that's when the reconnect is really "finished". In that case, yes that information is probably relevant.